ml_dtypes/_src/custom_float.h

```diff
@@ -876,17 +876,13 @@ bool RegisterFloatDtype(PyObject* numpy, bool* already_registered = nullptr) {
   // assumes that the other module has a sufficiently complete bfloat16
   // implementation. The only known NumPy bfloat16 extension at the time of
   // writing is this one (distributed in TF and JAX).
-  // TODO(phawkins): distribute the bfloat16 extension as its own pip package,
```

Comment:
I'm not sure this is needed any more! We used to do this to coexist with TensorFlow. But TensorFlow's current release now uses the definition from ml_dtypes, as do JAX and TensorStore.

I think we can just unconditionally register our type, and not try to interoperate with any other bfloat16 types. That is considerably safer, since the other types may be broken.

Reply:
OK, removed